exercises/reverse-string/canonical-schema.json

```diff
@@ -0,0 +1,54 @@
+{
+  "exercise": "reverse-string",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "empty string",
+      "property": "reverse-string",
+      "input": "",
+      "expected": ""
+    },
+       {
+      "description": "reverse a single character string",
+      "property": "reverse-string",
+      "input": "e",
+      "expected": "e"
+    },
+    {
+      "description": "reverse a string",
+      "property": "reverse-string",
+      "input": "robot",
+      "expected": "tobor"
+    },
+    {
+      "description": "reverse a string consisting of a single lowercase letter",
+      "property": "reverse-string",
+      "input": "WOLf",
+      "expected": "fLOW"
+    },
+    {
+      "description": "reverse a string having more than a single lowercase letter",
```

Comment:
This also describes the third test?

Reply:
When you are implementing a solution to this problem, you should be able to read the description and know from that what new aspect of the problem the test covers.
"a string having more than a single lowercase letter" has already been tested in the third test: "robot"

So either this test is redundant and should be removed because it is testing something that has already been tested.
OR 
This test needs a description that describes why it needs to test this particular input.
Why is "DEssERTS" useful to test?

Reply:
Perhaps you have a good example in one of the exercises that clearly shows good JSON and description files. I want to make sure we are on same page and not talking in circles for this simple problem. I know I'm new to this, so this will take alittle longer the first time around.